README.md

```diff
@@ -496,6 +496,8 @@ try {
 
 - Exporting internal classes other than `stdClass` and `Closure` is currently not supported. `VarExporter` will throw an `ExportException` if it finds one.
```

Comment:
Please just change this sentence to "other than `stdClass`, `Closure`, `DateTime` and `DateTimeImmutable`". According to php.net, the only other class that implements `__set_state()` is [MongoId](https://www.php.net/manual/en/mongoid.set-state), and is from a deprecated extension; so we can just assume that only the DateTime classes are concerned.

Reply:
The official documentation at php.net seems to be incomplete. The phostorm-stubs state that also other date related object implement an `__set_state()` method. An example of this is `DateTimeZone` ([php docs](https://www.php.net/manual/en/class.datetimezone.php), [stubs](https://github.com/JetBrains/phpstorm-stubs/blob/master/date/date_c.php#L581)), I've tested it here at 3v4l, which shows that the method seems to be implemented since php 5.5: https://3v4l.org/WFHj1

Reply:
Makes sense. A quick survey using reflection on `get_declared_classes()` gives me this list:

- DateTime
- DateTimeImmutable
- DateTimeZone
- DateInterval
- DatePeriod

Let's phrase it like this then:

> Exporting internal classes other than `stdClass`, `Closure`, and classes implementing `__set_state()` (most notably DateTime classes) is currently not supported. `VarExporter` will throw an `ExportException` if it finds one.

Reply:
A'right, just tested this myself too, and I agree and will change the paragraph accordingly.

https://3v4l.org/k7dYe